src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -219,6 +219,25 @@ struct SpacetimeMetricCompute : SpacetimeMetric<SpatialDim, Frame, DataType>,
       &spacetime_metric<SpatialDim, Frame, DataType>;
 };
 
+/*!
+ * \brief Compute item for inverse spacetime metric \f$\psi^{ab}\f$
```

Comment:
We are standardizing to \f$g^{ab}\f$ for inverses spacetime metric, \f$\alpha\f$ for lapse, and \f$\beta\f$ for shift. Might as well make this PR consistent with that. See #1494